### DISCUSSION TITLE
Can we improve our project infrastructure organization?

### DISCUSSION THREAD

**Discussion by kmpaul** [ProjectPythia/projectpythia.github.io#171] [Ideas]
I recently discovered [QuantEcon](https://quantecon.org).  The QuantEcon project is all about providing open source interactive educational material for quantitative economics.  They have examples, lectures, etc.  And their website appears to be a landing page (https://quantecon.org) and a collection of Jupyter Books.  Their content is *vast*!  And the more I look at their codebase, the more impressed I become with their:

- organizational structure for content (landing pages and a library of books and how they separate them across GitHub repos), and
- extensible Jupyter Book Theme (https://github.com/quantecon/quantecon-book-theme).

I think that it might benefit Project Pythia to learn about how their content is organized and how we can leverage their organizational structure and theme.

@ProjectPythia/core @ProjectPythia/infrastructure @ProjectPythia/education 

Thoughts?

**Comment by ktyle** [comment]
Yes @kmpaul , they look to be one of the most active deployers of Jupyterbook, as well as other Executable Books Project efforts! Coincidentally, I am also browsing their content this week, as I integrate Project Pythia content into my course's Jupyterbook (which I will talk about in tomorrow's Education call).
